tests/cli/test_cmds_local_run.py

```diff
@@ -1959,3 +1960,352 @@ def test_missing_volumes_skipped(mock_exists):
         )
         args, kwargs = mock_run_docker_container.call_args
         assert kwargs["volumes"] == []
+
+
+@mock.patch("paasta_tools.cli.cmds.local_run.pick_random_port", autospec=True)
+@mock.patch("paasta_tools.cli.cmds.local_run.get_docker_run_cmd", autospec=True)
+@mock.patch("paasta_tools.cli.cmds.local_run.execlpe", autospec=True)
+@mock.patch(
+    "paasta_tools.cli.cmds.local_run._run",
+    autospec=True,
+    return_value=(0, "fake _run output"),
+)
+@mock.patch("paasta_tools.cli.cmds.local_run.get_container_id", autospec=True)
+@mock.patch(
+    "paasta_tools.cli.cmds.local_run.get_healthcheck_for_instance",
+    autospec=True,
+    return_value=("fake_healthcheck_mode", "fake_healthcheck_uri"),
+)
+@mock.patch("paasta_tools.cli.cmds.local_run.decrypt_secret_volumes", autospec=True)
+@mock.patch("paasta_tools.cli.cmds.local_run.shutil", autospec=True)
+@mock.patch(
+    "paasta_tools.cli.cmds.local_run.open",
+    new_callable=mock.mock_open(),
+    autospec=None,
+)
+@mock.patch("os.makedirs", autospec=True)
+def test_run_docker_container_secret_volumes(
+    mock_os_makedirs,
+    mock_open,
+    mock_shutil,
+    mock_decrypt_secret_volumes,
+    mock_get_healthcheck_for_instance,
+    mock_get_container_id,
+    mock_run,
+    mock_execlpe,
+    mock_get_docker_run_cmd,
+    mock_pick_random_port,
+):
+    mock_pick_random_port.return_value = 666
```

Comment:
should this and mock_decrypt_secret_volumes have their return values set at the decorator level to be consistent with how we mocked `paasta_tools.cli.cmds.local_run._run`?

Reply:
(same would apply in the other tests if we take any action here)